jenkins/shared-library/vars/defaultCiPipeline.groovy

```diff
@@ -185,16 +186,16 @@ void call(Closure body) {
 				stages {
 					stage('publish RC') {
 						when {
-							anyOf {
-								branch env.DEV_BRANCH
-								allOf {
+							allOf {
```

Comment:
since packages with same version can't be published multiple times, i think we only want to publish on manual builds, else we'll get lots of build problems if we don't auto increment  (and i don't see good reason for that).

unsure if we should restrict test/alpha publishes from dev branch